### PR TITLE
changing caption in alueib

### PR DESCRIPTION
**Pull request by alueib**
critical change for caption change alueib #1

@TsnC5: please review and approve

### CODE REVIEW

File: _posts/0000-01-02-alueib.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![cherryontop-o-cat](https://octodex.github.com/images/cherryontop-o-cat.png)
 {: .center}
 
-Caption here
+Caption here, Caption there, Caption everywhere
```
Comment (alueib) [inline _posts/0000-01-02-alueib.md]: hurray

Comment (Sbruk) [reply _posts/0000-01-02-alueib.md]: NullPointerException?

Comment (alueib) [reply _posts/0000-01-02-alueib.md]: oh no, it must be your config
cannt reproduce

Comment (APBEI) [inline _posts/0000-01-02-alueib.md]: it's a great change

Comment (Sbruk) [reply _posts/0000-01-02-alueib.md]: Looks good then!

**Review by APBEI — APPROVED** [review]

**Review by Sbruk — APPROVED** [review]
Go go go!

File: _posts/0000-01-02-alueib.md (hunk 1 above)
Comment (ImSo06) [inline _posts/0000-01-02-alueib.md]: 👍

**Review by ImSo06 — APPROVED** [review]

File: _posts/0000-01-02-alueib.md (hunk 1 above)
Comment (alueib) [reply _posts/0000-01-02-alueib.md]: thank you great review 🥇

File: _posts/0000-01-02-alueib.md [hunk 2]
```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![cherryontop-o-cat](https://octodex.github.com/images/cherryontop-o-cat.png)
 {: .center}
 
-Caption here
+Caption here, Caption there, Captions everywhere
```
Comment (alueib) [inline _posts/0000-01-02-alueib.md:9]: good

**Review by alueib — COMMENTED** [review]
done!

**Pull request merged**